management/client.go

```diff
@@ -84,6 +84,14 @@ func (c *Client) String() string {
 	return string(b)
 }
 
+// MarshalJSON encodes the client structure to JSON excluding the signing_keys field
+func (c *Client) MarshalJSON()([]byte, error) {
+	copy := *c
+	copy.SigningKeys = nil
+	// Marshal copy here, not &copy, to avoid recursion
+	return json.Marshal(copy)
+}
+
```

Comment:
@harryrose why do this in particular for signing keys? There are many other fields that are read-only, but we don't actively omit them during serialization but rather leave it up to users to do so.

Could we omit the `MarshalJSON` and let users set `SigningKeys` to nil when creating/updating?